buildkite/build_project.sh

```diff
@@ -295,11 +298,19 @@ case "$REPO_FULL_NAME" in
         export TMPDIR="$TMP"
         rm -rf "$TMP" && mkdir -p "$TMP"
         # patch makefile which requires gdb 8 - see https://github.com/dlang/ci/pull/301
-        sed "s/TESTS+=rt_trap_exceptions_drt_gdb//" -i dmd/druntime/test/exceptions/Makefile
+        if [ -f dmd/druntime/test/exceptions/Makefile ]; then
+            sed "s/TESTS+=rt_trap_exceptions_drt_gdb//" -i dmd/druntime/test/exceptions/Makefile
+        elif [ -f druntime/test/exceptions/Makefile ]; then
+            sed "s/TESTS+=rt_trap_exceptions_drt_gdb//" -i druntime/test/exceptions/Makefile
+        fi
```

Comment:
I can't say why you needed to revert it all, but on a second look, there's a thinko here, where I should have kept in the if condition.

Reply:
Because I saw the CI checking out `master` and building it.
I figured best way forward was to:
1) Keep support for building the old scheme
2) Add support for the new scheme
3) Remove `druntime` as a project we test